Tests/test_file_libtiff.py

```diff
@@ -263,7 +260,7 @@ def test_custom_metadata(self):
                     tc(4.25, TiffTags.FLOAT, True),
                     tc(4.25, TiffTags.DOUBLE, True),
                     tc("custom tag value", TiffTags.ASCII, True),
-                    tc(u"custom tag value", TiffTags.ASCII, True),
```

Comment:
Looks like this line is now useless, as it is identical to the previous line.

Reply:
Thanks, removed in 0500dcb0083ba9e1a80608f06ba430246b59507c.